Hospital/src/location/Department.java

```diff
@@ -0,0 +1,215 @@
+package location;
+
+import customExceptions.InvalidFloorNumberException;
+import interfaces.IContainsPersonel;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import person.Doctor;
+import person.Nurse;
+import person.Patient;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class Department implements IContainsPersonel {
+    private static final Logger logger = LogManager.getLogger("console logger");
```

Comment:
please change it for all classes
```suggestion
public class Department implements IContainsPersonel {
    private static final Logger logger = LogManager.getLogger(Department.class);
```